llama_index/embeddings/openai.py

```diff
@@ -282,6 +282,7 @@ def __init__(
 
         if "model_name" in kwargs:
             model_name = kwargs.pop("model_name")
+            self._query_engine = self._text_engine = model_name
```

Comment:
so in this case query engine and text engine using the same emb model?

Reply:
OpenAI already uses a single model name (if you check the dicts above, the literals are all the same for ada). 

I guess technically we should probably allow some way to set a different model for each engine. Maybe some optional additional kwarg?

Reply:
got it. for openai emb I think this should be okay.
agree with you. we probably can pass kwarg to separate text/query emb for other models